lib/msal-core/src/Storage.ts

```diff
@@ -175,4 +175,20 @@ export class Storage {// Singleton
         this.setItemCookie(Constants.loginRequest, "", -1);
         this.setItemCookie(Constants.stateAcquireToken, "", -1);
     }
+
+    /**
+     * Create acquireTokenUserKey to cache user object
+     * acquireTokenUserKey = Constants.acquireTokenUser + Constants.resourceDelimeter + userId + Constants.resourceDelimeter + authenticationRequest.state;
+     */
+    static generateATUserKey(userId: any, state: string) {
```

Comment:
can we change the method name, so that it's not cryptic? something like generateCacheKeyForAcquireToken / generateAcquireTokenUserKey

Reply:
Sure manoj. Will do.